backend/README.md

```diff
@@ -0,0 +1,47 @@
+## Getting Started
+
+This README will take you through the process to setup the backend folder locally.
+
+### Prerequisites
+
+The following technologies are required to run the server and use it locally.
+
+- nodejs
```

Comment:
embed the links from where the following technlogies can be installed

Reply:
and I think the command is mongo not ```mongod``` to start MongoDB shell is it ?

Reply:
Yes ```mongo``` is used to start the mongo shell, but here we need to start a local mongodb server not the shell, and most installation procedures make you set ```mongod``` as an alias to start the mongodb server